tools/mason/MasonExternal.chpl

```diff
@@ -298,7 +298,7 @@ proc getSpkgInfo(spec: string, ref dependencies: list(string)): unmanaged Toml t
     if spkgInstalled(spec) {      
       const spkgPath = getSpkgPath(spec);
       const libs = joinPath(spkgPath, "lib");
-      const include = joinPath(spkgPath, "include");
+      const include_ = joinPath(spkgPath, "include");
```

Comment:
`incPath`?  `includePath`?  (just to use a slightly less lame identifier).  (And ditto in the other mason file?)

Reply:
Done